spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/ProviderRoleMapping.java

```diff
@@ -37,6 +37,7 @@ public class ProviderRoleMapping {
 	private String rolePrefix = "ROLE_";
 	private String groupClaim = "roles";
 	private boolean mapOauthScopes = false;
+	private boolean parsePathParts = true;
```

Comment:
I wonder if there is another name we could use that describes the scope in this case rather than the action we do not want to take on it? 

This one will work but it does feel like one must know when to parse and not to parse. Wdyt? 

P.S. I don't have a better name :)

Reply:
I renamed the property